IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -158,24 +161,23 @@ - (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(__unused NSEr
     _realm = json[MSID_REALM_CACHE_KEY];
     _environment = json[MSID_ENVIRONMENT_CACHE_KEY];
     _expiresOn = [NSDate msidDateFromTimeStamp:json[MSID_EXPIRES_ON_CACHE_KEY]];
+    _extendedExpiresOn = [NSDate msidDateFromTimeStamp:json[MSID_EXTENDED_EXPIRES_ON_CACHE_KEY]];
```

Comment:
Legacy token cache item serialization also needs to be updated, where it expects it to be in the additional info. Otherwise, looking good.

Reply:
👍